gradle-android-command-plugin/src/test/groovy/com/novoda/gradle/command/AndroidCommandPluginExtensionTest.groovy

```diff
@@ -27,7 +22,7 @@ class AndroidCommandPluginExtensionTest extends GroovyTestCase {
     private static AndroidCommandPluginExtension createExtension() {
 	def projectDir = new File('..')
         def project = ProjectBuilder.builder().withProjectDir(projectDir).build()
-        def extension = new AndroidCommandPluginExtension(project)
+        def extension = new AndroidCommandPluginExtension(project, null)
```

Comment:
Might use some dummy string here instead of `null`, like "addFolder" or the likes.